mailet/icalendar/pom.xml

```diff
@@ -109,6 +113,17 @@
         <dependency>
             <groupId>org.mnode.ical4j</groupId>
             <artifactId>ical4j</artifactId>
+            <version>3.0.0</version>
+        </dependency>
+        <dependency>
+            <groupId>javax.cache</groupId>
+            <artifactId>cache-api</artifactId>
+            <version>${javax.cache.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.jsr107.ri</groupId>
+            <artifactId>cache-ri-impl</artifactId>
```

Comment:
this implementation is not for production use (see https://github.com/jsr107/jsr107spec )

Reply:
TBH, I'd rather not include yet-another dependency just for that, please read https://github.com/ical4j/ical4j/issues/195 , there's a solution to avoid providing a jcache implementation